README.md

```diff
@@ -1,5 +1,7 @@
 # Castle Project
 
+[![Join the chat at https://gitter.im/castleproject/all](https://badges.gitter.im/castleproject/all.svg)](https://gitter.im/castleproject/all?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
As with the one for Castle Core I think we'd be best putting it under a heading that is about contributing, one of the existing headings here probably suit, otherwise we risk people using it as their first point of call for questions.